torch/_inductor/config.py

```diff
@@ -1035,6 +1035,12 @@ class simplefsdp:
     enable_bucket = False
```

Comment:
remove

Reply:
We cannot remove `enable_bucket`, because it was also called [LINK](https://github.com/tianyu-l/pytorch_intern24/blob/211847de3c1c3d6cbd299e14a001b794eabf2a2d/torch/_inductor/ir.py#L5994) [LINK](https://github.com/tianyu-l/pytorch_intern24/blob/211847de3c1c3d6cbd299e14a001b794eabf2a2d/torch/_inductor/ir.py#L7097).


We can disable user API, and do an additional `enable_bucket` config in the scheduler.py

Reply:
I updated it as bucket_mode, as we discussed offline